src/Application/Features/Reviewer/Query/GetAllReviewer/GetAllReviewerCommandHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MRA.Jobs.Application.Contracts.Reviewer.Queries;
+using MRA.Jobs.Application.Contracts.Reviewer.Response;
+
+namespace MRA.Jobs.Application.Features.Reviewer.Query.GetAllReviewer;
+
+public class GetAllReviewerCommandHandler : IRequestHandler<GetAllReviewerQuery, ReviewerListDTO>
```

Comment:
Not completed yet. Seems you did not finish this handler

Reply:
ok